README.md

```diff
@@ -68,7 +61,13 @@ For complete coverage details, open `coverage/lcov-report/index.html`.
 
 The tests and coverage covers the Node.js module; it does not cover the command-line interface (which is tiny) or the built JavaScript library (which is the same code just ran through browserify).
 
-### Building the JavaScript library
+## Experimental JavaScript library
+
+### Usage
+
+Include `build/gltf-pipeline.js` it with a `script` tag.  For a simple example, see [build/index.html](build/index.html).
```

Comment:
`build/gltf-pipeline.js` is empty now, but even before was pretty out of date. And `index.js` is a lot smaller than it used to be, so `index.html` won't work. Do we still want to include these instructions?